src/components/Hero/index.jsx

```diff
@@ -1,9 +1,12 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-import edXLogoForBanner from '../../images/edx-logo-for-banner.png';
 import './Hero.scss';
 
+import { configuration } from '../../config';
+
+const edxWhiteSemiTransparentLogo = configuration.LOGO_WHITE_URL;
```

Comment:
nit: It feels a bit odd to create this variable outside of the component. Any reason not to reference `configuration.LOGO_WHITE_URL` directly on line 18?